test_scripts/Smoke/API/007_Alert_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,196 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: Alert
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [Alert] SUCCESS: request with UI portion and TTSChunks
+--
+-- Description:
+-- Mobile application sends valid Alert request with UI-related-params & with TTSChunks
+-- and gets SUCCESS resultCode to both UI.Alert and TTS.Speak from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests Alert with UI-related-params & with TTSChunks
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if Alert is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the UI.Alert part of request with allowed parameters to HMI
+-- SDL transfers the TTS.Speak part of request with allowed parameters to HMI
+-- SDL receives UI.Alert part of response from HMI with "SUCCESS" result code
+-- SDL receives TTS.Speak part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local putFileParams = {
+	requestParams = {
+	    syncFileName = 'icon.png',
+	    fileType = "GRAPHIC_PNG",
+	    persistentFile = false,
+	    systemFile = false
+	},
+	filePath = "files/icon.png"
+}
+
+local requestParams = {
+	alertText1 = "alertText1",
+	alertText2 = "alertText2",
+	alertText3 = "alertText3",
+	ttsChunks = {
+		{
+			text = "TTSChunk",
+			type = "TEXT",
+		}
+	},
+	duration = 3000,
```

Comment:
@AKalinich-Luxoft Since 'duration' and 'softButtons' parameters are mutually exclusive. Please split one test step into a two ones.

Reply:
@dboltovskyi done in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1716/commits/35de742bd2e2b1a7ba97057dfb39fb9a225c97ef